libraries/helpers.rb

```diff
@@ -32,7 +32,7 @@ def vagrant_platform_package(vers = nil)
     case node['platform_family']
     when 'debian'
       "vagrant_#{vers}_x86_64.deb"
-    when 'redhat', 'fedora'
```

Comment:
With this change, we can avoid having to inspect both `node['os']` and then`node['platform_family']` for the linux case.